packages/blade/src/components/Typography/Display/Display.stories.tsx

```diff
@@ -0,0 +1,145 @@
+      }}
+    >
+      <StorybookTitle>Usage</StorybookTitle>
+      <Sandbox>
+        {`
+          import { Display } from '@razorpay/blade/components';
+
+          function App(): React.ReactElement {
+            return (
+              <Display size="large">Blade by Razorpay</Display>
+            )
+          }
+
+          export default App;
+        `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+const DisplayStoryMeta: Meta<DisplayProps> = {
+  title: 'Components/Typography/Display',
+  component: DisplayComponent,
+  args: {
+    size: 'small',
+    type: 'normal',
+    children: 'Power your finance, grow your business',
+    contrast: 'low',
+    as: undefined,
+  },
+  argTypes: {
+    size: {
+      options: ['small', 'medium', 'large', 'xlarge'],
+      control: {
+        type: 'radio',
+      },
+      table: {
+        type: {
+          summary: '"small" | "medium" | "large" | "xlarge"',
+        },
+      },
+    },
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: () => <Page />,
+    },
+  },
+};
+
+const DisplayTemplate: ComponentStory<typeof DisplayComponent> = (args) => {
+  return <DisplayComponent {...args}>{args.children}</DisplayComponent>;
+};
+
+export default DisplayStoryMeta;
+export const Display = DisplayTemplate.bind({});
+export const WithColor = DisplayTemplate.bind({});
+WithColor.args = {
+  color: 'brand.primary.500',
+};
+
+const Sup = isReactNative() ? DisplayComponent : 'sup';
+const WithMixedColorsTemplate: ComponentStory<typeof DisplayComponent> = (args) => {
+  return (
+    <Box>
+      <DisplayComponent {...args}>
+        Supercharge your business with the all‑powerful{' '}
+        <DisplayComponent {...args} as="span" color="brand.primary.500">
+          Payment Gateway
+        </DisplayComponent>
+      </DisplayComponent>
+      <DisplayComponent marginTop="spacing.5" {...args}>
+        Start accepting{' '}
+        <DisplayComponent {...args} as="span" color="feedback.text.information.lowContrast">
+          payments
+        </DisplayComponent>{' '}
+        at just 2% <Sup>*</Sup>
+      </DisplayComponent>
+    </Box>
+  );
+};
+
+export const WithMixedColors = WithMixedColorsTemplate.bind({});
+
+const AsPropTemplate: ComponentStory<typeof DisplayComponent> = (args) => {
+  return (
+    <Box>
+      <Text>
+        By default{' '}
+        <Text as="span" weight="bold">
+          Display
+        </Text>{' '}
+        component automatically renders the respective <Code size="medium">h*</Code> tag based on
+        the{' '}
+        <Text as="span" weight="bold">
+          size prop
+        </Text>{' '}
+        passed
+      </Text>
+      <List>
+        <ListItem>small: h3</ListItem>
+        <ListItem>medium: h2</ListItem>
+        <ListItem>large: h1</ListItem>
+        <ListItem>xlarge: h1</ListItem>
```

Comment:
By default display should always render `h1`, because it's the largest of them all, even it's small size is same has the Title.

Reply:
This makes sense. @kamleshchandnani I've updated it to be `h1` always. If `Title large` and `Display small` exists on the same screen, the Display should not be `h3`.